Source/CarthageKit/Project.swift

```diff
@@ -117,6 +118,10 @@ public enum ProjectEvent {
 	/// Building the project is being skipped, since the project is not sharing
 	/// any framework schemes.
 	case skippedBuilding(ProjectIdentifier, String)
+
+	/// Installing of a binary framework is being skipped because of an inability
+	/// to verify that it was built with a compatible Swift version.
+	case skippedInstallingBinaries(ProjectIdentifier, String)
```

Comment:
I was trying to make this event more general than just the incompatible swift versions event, incase other reasons for skipping a binary installation were added. I'd love to add the labels, I was just copying the local coding style as suggesting in `CONTRIBUTING`. Will change.